CRM/Core/BAO/Setting.php

```diff
@@ -199,6 +199,10 @@ public static function setItems(&$params, $domains = NULL) {
     $fieldsToSet = self::validateSettingsInput($params, $fields);
 
     foreach ($fieldsToSet as $settingField => &$settingValue) {
+      if (empty($fields['values'][$settingField])) {
+        Civi::log()->warning('Deprcated Path There is a setting (' . $settingField . ') not correctly defined. You may see unpredictabilitiy due to this, CRM_Core_Setting::setItems', array('civi.tag' => 'deprecated'));
```

Comment:
@eileenmcnaughton is there anywhere that we can put things like this as a "todo list" for removal in future? Looking at a single case it doesn't seem too bad, but in bulk we would end up with a very hard-to-read codebase that offers warnings like this that only ever increase.